src/stores/timed.rs

```diff
@@ -52,9 +47,33 @@ impl<K: Hash + Eq, V> TimedCache<K, V> {
             hits: 0,
             misses: 0,
             initial_capacity: Some(size),
+            refresh: false,
+        }
+    }
+
+    /// Creates a new `TimedCache` with a specified lifespan which
+    /// refreshes the ttl when the entry is retreived
+    pub fn with_lifespan_and_refresh(seconds: u64, refresh: bool) -> TimedCache<K, V> {
```

Comment:
since refresh defaults to false, can it be implied that calling `with_lifespan_and_refresh(60)` would "enable" refresh?

Reply:
Yes, it could. I would prefer not to do It, maybe the value for refresh comes from some other place and is not constant.